### ISSUE TITLE
The description of the relationship between an object and a TTP is not displayed

### ISSUE THREAD

**Issue by lightw1s3** [LuatixHQ/opencti#8981]
## Description

¡Hello!, I have detected a change in the behavior when creating a relationship between any entity and an 'Attack Pattern'. 
Currently, when an attack pattern is introduced from the entity (for example a Threat Actor or a Campaign) despite writing a description in the relationship itself identifying the detail of the use of such TTP, the description that comes out in the “Kill Chain View” window is the one of the TTP object itself and not the one written by the analyst.
This did not happen in version 6.2.0 (version I had before, so it is the one I have identified), in the steps section I will show you examples.

## Environment

1. OS: docker.
2. OpenCTI version: 6.3.8
3. OpenCTI client: frontend
4. Other details: It has been checked by deploying version 6.3.11 and the operation is exactly the same.

## Reproducible Steps // Actual output -> + Screenshots

Steps to create the smallest reproducible scenario for version 6.3.8:
1. Create a Threat Actors (group) named FIN8 and access 'Knowledge'. In turn, access the Attack patterns section.

![Image](https://github.com/user-attachments/assets/48f9283b-7f5b-4c0c-8a3f-0eb3445a6aa8)

2. A new TTP is added, in this case the one shown in the image, and in the description field an example text is written to simulate the explanation of how an actor performs the TTP.

![Image](https://github.com/user-attachments/assets/b8eebaa8-bfb5-4403-b9e6-d793e2adf2a2)

![Image](https://github.com/user-attachments/assets/fec56fc9-9b3b-4f5a-988d-1d29ffffa6fd)

3. The creation of the TTP is displayed and, as can be seen, the TTP definition itself appears in the description field (Moreover it is not shown in the correct kill chain but I will report this in another bug :( )

![Image](https://github.com/user-attachments/assets/a83ebb6f-810f-47d7-9bc3-48ef0f037a70)

## Expected Output -> + Screenshots

Steps to create the smallest reproducible scenario for version 6.2.0:

1. Create a Threat Actors (group) called Issue OpenCTI and go to 'Knowledge'. In turn, access the Attack patterns section, in this case it is empty.

![Image](https://github.com/user-attachments/assets/1218395d-518d-42fb-915c-2ef4e0a37b98)

2. A new TTP is added, in this case the one shown in the image, and in the description field the same example text is written to simulate the explanation of how an actor performs the TTP.

![Image](https://github.com/user-attachments/assets/40185aab-536d-44b7-926d-899fd85eaf3b)

![Image](https://github.com/user-attachments/assets/55fca28c-d0e4-469c-bc02-6d537811d39a)

3. Once the same TTP has been created, it is possible to check how the result is different by showing the description of the relationship and not the description of the TTP object.

![Image](https://github.com/user-attachments/assets/8699e9e8-d433-431a-bb89-9fe20becb415)

**Comment by nino-filigran** [comment]
@lightw1s3 indeed, we've noticed this change: when working on this view, a unwated side effect has been created which is that the perspective behind these screens have changed from relation to entity as you have noticed. We have a task to adress this change: https://github.com/OpenCTI-Platform/opencti/issues/8835

**Comment by lightw1s3** [comment]
Great! Sorry for the repetition, thank you very much for your answer.

**Comment by lightw1s3** [comment]
I just deployed version 6.4.9 which fixes this Issue and, it seems that it is not fixed. The description of the TTPs are still the ones found in the Mitre object and not the ones provided in the relation.

I don't know if it was a mistake or if you have it planned as you said to fix it with Issue #8835 

Sorry for the question but the truth is that it is a solution that we are looking forward to come out :) 

Thanks, 

![Image](https://github.com/user-attachments/assets/cac0c4ba-0ff8-4d80-b29f-fe49d6a8a6a2)

![Image](https://github.com/user-attachments/assets/99a1a93b-48fc-48eb-a797-e567055d7487)

**Comment by nino-filigran** [comment]
@lightw1s3 ITt's not a bug: as you have pointed it out, all the TTP views except the TTP relation view display the **entity** view of the TTP. As a result, it is "normal" that you don't see the relation description. 
It's a choice made by design, since overall we had a push from the community to display entity rather than relation.

Why do you need to visualize a relation description there?

**Comment by lightw1s3** [comment]
Hello!
Ok I understand that it is not a mistake. I comment you because it is much more useful to see in this case here the description of the relationship. When we model an actor or an IS, Campaign related to it, our technical team doesn't really see so useful to know which TTP has been used but the HOW the actor uses it. For example: It uses the TTP T1059.001 (Powershell) but the most interesting thing is not to see in this view the definition of the TTP but the description of the relationship because there is the real detail of how the actor or the object would use it. For example, I show you an image of the content of the description of a relationship.
Therefore, as an analyst (where I can also speak for my team) it is more useful to have a view that shows all the TTPs together with the description of how the actor uses it at a glance (and not having to access one by one to see how it is used by the actor).

<img width="804" height="272" alt="Image" src="https://github.com/user-attachments/assets/f4d7c5b5-eded-4274-91db-c34200eea6ac" />

**Comment by alice-debra** [comment]
Hello @lightw1s3,

Since this is no longer a bug but a design choice, I suggest you open a feature request to track your request. Would that be okay with you?

Kind regards,

Alice
